### PR TITLE
[BugFix] Table using light schema change will lost bloom filter in rowset data.

### PR DESCRIPTION
**Pull request by sevev**
When we create a table using light schema change, we will rebuild tablet schema according to the schema in FE. But we lost the bloom filter property in the column. So we will lost the bloom filter in the rowset data.

## What type of PR is this:

- [x] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [x] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 0 / 7 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/storage/tablet_schema.cpp|0|7|00.00%|[484, 485, 486, 487, 505, 506, 507]|

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by decster — DISMISSED** [review]

File: be/src/storage/olap_meta_reader.cpp [hunk 1]
```diff
@@ -95,7 +95,7 @@ Status OlapMetaReader::_build_collect_context(const OlapMetaReaderParams& read_p
         }
         _has_count_agg |= (collect_field == "count");
     }
-    _collect_context.seg_collecter_params.tablet_schema = read_params.tablet->tablet_schema();
+    _collect_context.seg_collecter_params.tablet_schema = read_params.tablet_schema;
```
Comment (chaoyli) [inline be/src/storage/olap_meta_reader.cpp:98]: 1. If FE doesn't record this infos, how it show the `bloom filter` when `show create table`?
2. If the bloom filter has this issue, does the bitmap index has this issue too?

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 0 / 8 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/storage/olap_meta_reader.cpp|0|1|00.00%|[98]|
|:large_blue_circle:|src/storage/tablet_schema.cpp|0|7|00.00%|[484, 485, 486, 487, 505, 506, 507]|

File: be/src/storage/olap_meta_reader.cpp (hunk 1 above)
Comment (sevev) [reply be/src/storage/olap_meta_reader.cpp:98]: > 1. If FE doesn't record this infos, how it show the `bloom filter` when `show create table`?
> 2. If the bloom filter has this issue, does the bitmap index has this issue too?

1. The attributes of most columns are primarily stored within the columns themselves, except for the property of whether to create a Bloom filter. The FE maintains the names of columns with Bloom filters. As a result, when you execute a "show create table" command, you can see this information. However, the FE loses the Bloom filter property when transmitting column information to the BE(in function `Column.toThrift()`)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]

:heart_eyes: **pass** : 6 / 6 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/planner/OlapScanNode.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/planner/OlapTableSink.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/catalog/Column.java|2|2|100.00%|[]|

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 0 / 8 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/storage/olap_meta_reader.cpp|0|1|00.00%|[98]|
|:large_blue_circle:|src/storage/tablet_schema.cpp|0|7|00.00%|[484, 485, 486, 487, 505, 506, 507]|

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]

:disappointed: **fail** : 0 / 1 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|src/storage/olap_meta_reader.cpp|0|1|00.00%|[98]|

**Review by chaoyli — APPROVED** [review]

**Review by decster — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.2

**Pull request merged**